index.html

```diff
@@ -11,69 +11,338 @@
 </head>
 
 <body>
```

Comment:
- [ ] Your design appears aligned to the _left_. Kindly ensure that the contents of your page is **properly centered**, as seen in the [design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468).

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/177012159-7fd4c681-06c8-4b1d-949a-d6a3850cd116.png)


**The actual design:**
![image](https://user-images.githubusercontent.com/12745474/177012238-1fbb958c-86f9-46de-9766-e6c6d96d060c.png)